src/gen_rules.ml

```diff
@@ -774,11 +774,45 @@ module Gen(P : Params) = struct
     | None -> []
     | Some name -> ["--cookie"; sprintf "library-name=%S" name]
 
+  (* Generate rules for the reason modules in [modules] and return a
+     a new module with only OCaml sources *)
+  let setup_reason_rules ~dir (m : Module.t) =
+    let refmt =
+      match Artifacts.binary "refmt" with
```

Comment:
Just to confirm, will this work if `refmt` is a symlink? It very well might be. Nice touch with the hint to install via opam.